modules/test/playwright/tests/osb-faro-web/page.spec.ts

```diff
@@ -128,3 +128,76 @@ test('shows outside pages in path analysis', async ({apiHelpers, page}) => {
 
 	await apiHelpers.jsonWebServicesLayout.deleteLayout(String(sitePage.id));
 });
+
+test('shows tracked pages in path analysis', async ({apiHelpers, page}) => {
+	const pageTitle1 = 'My Page 1';
+
+	const sitePage1 = await createSitePage(apiHelpers, pageTitle1);
+
+	const pageTitle2 = 'My Page 2';
+
+	const sitePage2 = await createSitePage(apiHelpers, pageTitle2);
+
+	const channelName = 'My Property - ' + getRandomString();
+
+	await syncAnalyticsCloud(page, channelName);
+
+	await page.waitForTimeout(10000);
+
+	await page.getByText(pageTitle1).first().click();
+
+	await page.getByText(pageTitle2).first().click();
+
+	await page.waitForTimeout(10000);
```

Comment:
Hey @rachaelkoestartyo, I think this pause should be placed after you generate the interaction with the page to guarantee that the event was sent. I believe this pause after the connection is not necessary. What do you think?

```suggestion
	await page.getByText(pageTitle1).first().click();

	await page.waitForTimeout(10000);

	await page.getByText(pageTitle2).first().click();

	await page.waitForTimeout(10000);
```